dotcom-rendering/playwright.config.ts

```diff
@@ -0,0 +1,82 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
Leftover?

Also, [reading `.env` is now supported since Node 20.6](https://nodejs.org/en/blog/release/v20.6.0)!

Reply:
This file is auto-generated by the Playwright setup. I left the commented out sections as is to demonstrate the configuration options. Happy to remove if we think it is just adding noise.

Reply:
Have removed any auto-generated but unused configuration